app/views/application/previewapplication/generalDetailsViewSection.scala.html

```diff
@@ -1,5 +1,27 @@
 @(generalDetails: connectors.ExchangeObjects.GeneralDetailsExchange)(edit: Html)
 
+@import _root_.forms.FastPassForm
+
+
+@displayFastPassSection(id:String, display:Boolean, heading: String, text: => String) = @{
+    if(display){
+        <h3 class="heading-small">{heading}</h3>
+        <p id={id}>{text}</p>
+    }
+}
+
+@getYesOrNo(value:Boolean) = @{
+    if(value) "Yes" else "No"
+}
+
+@getFastPassTypeDescription(fastPassType: String) = @{
+    FastPassForm.FastPassTypes.find(_._1 == fastPassType).map(_._2).getOrElse("")
```

Comment:
Because we're tying so many things into Seq[(String, String)] you end up having to write this rather than .get(fastPassType).getOrElse("") if you had a Map. I think we might need to attempt this form data refactor sooner rather than later